projects/npm-tools/packages/npm-scripts/contrib/prettier/prettier.sh

```diff
@@ -5,11 +5,27 @@
 # "liferay-npm-scripts prettier" subcommand to plugins.
 #
 
-# Auto-detect liferay-portal checkout.
+LIFERAY_NPM_SCRIPTS=""
+LIFERAY_PRETTIER=""
+
+# Auto-detect a liferay-portal or liferay-frontend-projects checkout.
```

Comment:
The following section is also pretty ugly. Not quite as bad as the VSCode one, but still not great.

![shaking-head](https://user-images.githubusercontent.com/7074/95591813-191a2900-0a48-11eb-94d5-8db2f3b73a4c.gif)